defaults/severities.yaml

```diff
@@ -64,6 +64,8 @@ KclStressEvent.start: NORMAL
 KclStressEvent.finish: NORMAL
 CassandraStressLogEvent.IOException: ERROR
 CassandraStressLogEvent.ConsistencyError: ERROR
+CassandraStressLogEvent.OperationOnKey: CRITICAL
+CassandraStressLogEvent.TooManyHintsInFlight: CRITICAL
```

Comment:
I have second thoughts about this one, I know that in some cases we have it stopping some stress and there isn't a solution at the moment.
Le'ts change set it here is ERROR and put a "TODO:" note to come back to it when we have more stable hinted handoff backpressure mechanism.

Reply:
@roydahan changed